doc/flatpak-run.xml

```diff
@@ -112,6 +112,14 @@
             Flatpak sets the environment variable <envar>FLATPAK_ID</envar> to the application
             ID of the running app.
         </para>
+        <para>
+            Flatpak also parses <filename>/usr/lib/os-release</filename> on the host, and for
+            each key-value pair sets an environment variable prefixed with service_host_,
+            as specified in the <ulink url="https://systemd.io/CONTAINER_INTERFACE">
+            Container Interface specification</ulink> and in the
+            <ulink url="https://www.freedesktop.org/software/systemd/man/os-release.html">
+            os-release specification</ulink>. Example: <envar>service_host_ID=debian</envar>
```

Comment:
This convention is not yet part of the referenced specification. I know you have a RFE open to *make* it part of the specification, but I don't think it's appropriate to merge this PR - particularly with this text included - until the systemd RFE has been accepted (which might lead to changes).

Reply:
Removed mentions to the spec. As with the inline comment in the source file, just wanted to present a view of how it would look like once it's done and dusted.

Reply:
Regarding merging, of course it's fine for me to wait until the discussion on the RFE resolves, no problem at all.